gradle/libs.versions.toml

```diff
@@ -91,6 +91,7 @@ node-plugin = "7.0.1"
 commons-cli = "1.2"
 sun-activation-version = "1.2.0"
 error-prone = "3.1.0"
+thrift = "0.12.0"
```

Comment:
why using `thrift`?

Reply:
Because HudiHMSBackend depends on HMS, so I introduce:
```
  implementation(libs.hive2.metastore) {
    exclude("*")
  }
```
and the runtime depends on thrift

Reply:
got it, another solution may not exclude thrift, it depends on you.